### PR TITLE
feat: Add $output params for geojson

### PR DESCRIPTION
**Pull request by mdartic**
Need to merge first #311

### CODE REVIEW

File: packages/engine/src/adapters/sql/index.ts [hunk 1]
```diff
@@ -657,12 +725,26 @@ export class SQLAdapter implements GenericAdapter {
...
+    /**
+     * Check if any data inserted is a geom one
+     */
+    const geomTables = currentModel.geomTables
+    Object.keys(data).forEach((dataKey) => {
+      if (geomTables.includes(dataKey)) {
+        // @ts-ignore
+        console.log('dataKey geom found ', dataKey, data?.[dataKey])
+        // @ts-ignore
+        data[dataKey] = raw(`ST_GeomFromText(?)::geometry`, [data[dataKey]])
```
Comment (Lonnytunes) [inline packages/engine/src/adapters/sql/index.ts:743]: Is `::geometry` really necessary? The call to `ST_GeomFromText()` is not sufficient?

:warning: Some `console.log()` calls still remain in this function.

File: packages/engine/src/adapters/sql/index.ts [hunk 2]
```diff
@@ -206,20 +233,32 @@ export class SQLAdapter implements GenericAdapter {
           })
         }
 
+        /**
+         * Does the current model have at least one geom column
+         */
+        static get hasGeomTable(): boolean {
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/engine/src/adapters/sql/index.ts]: Sorry, done !

File: packages/engine/src/adapters/sql/index.ts [hunk 3]
```diff
@@ -206,20 +233,30 @@ export class SQLAdapter implements GenericAdapter {
...
+          return t.columns.findIndex((c) => isGeometryColumn(c)) > -1
+        }
+        static get geomTables(): string[] {
+          return t.columns.filter((c) => isGeometryColumn(c)).map((c) => c.name)
+        }
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/engine/src/adapters/sql/index.ts]: done

File: packages/engine/src/adapters/sql/index.ts (hunk 1 above)
Comment (mdartic) [reply packages/engine/src/adapters/sql/index.ts]: You're right, I remove the `::geometry` cast.

**Pull request merged**